src/unittest/test_utilities.cpp

```diff
@@ -253,6 +255,15 @@ void TestUtilities::testUTF8()
 		== "the shovel dug a crumbly node!");
 }
 
+void TestUtilities::testRemoveEscapes()
+{
+	UASSERT(remove_escapes(L"abc\x1bXdef") == L"abcdef");
+	UASSERT(remove_escapes(L"abc\x1b(escaped)def") == L"abcdef");
+	UASSERT(remove_escapes(L"abc\x1b((escaped with parenthesis\\))def") == L"abcdef");
+	UASSERT(remove_escapes(L"abc\x1b(incomplete") == L"abc");
+	UASSERT(remove_escapes(L"escape at the end\x1b") == L"escape at the end");
+}
```

Comment:
How about a unit test that has a nested escape sequence, like \x1b(blah\x1b(asdf)blah)dffgd?